ui/app/store/actions.js

```diff
@@ -2458,6 +2458,16 @@ export function setConnectedStatusPopoverHasBeenShown() {
   }
 }
 
+export function setMobileWelcomeMessageHasBeenShown() {
+  return () => {
+    background.setMobileWelcomeMessageHasBeenShown((err) => {
```

Comment:
paying forward the advice @Gudahtt gave me recently... We should prefer the 'promisifiedBackground' for all new usages.

Reply:
Also if you're not using dispatch or setState here you don't have to return a function.

Reply:
We do not. This module's name (`actions`) is a bit of a misnomer - really it's a combination of two distinct things: action creators and calls to the background. Sometimes a background call is embedded within an action creator, but the functions that _just_ call the background don't need any wrapper function or any of the usual redux action creator boiler plate stuff.

On my "someday" list, once we finish replacing `background` with `promisifiedBackground` everywhere, I had planned to split this into two distinct modules (`actions.js` and `background.js`), so this distinction was more clear.